scanner/nftScanner.py

```diff
@@ -140,16 +187,30 @@ def start_NFT_scan(self, target_block):
             if(blockDiff < self.batch_size):
                 batchSize = blockDiff
 
+        # Update last scanned block
+        try:
+            self.session.query(NFTData).update({'last_scan_block': target_block})
```

Comment:
I think this code is updating every single last_scan_block column within the DB. Should not update all columns without any specification. There should probably only have 1 last_scan_block value in the database.

Reply:
I wasn't sure of the DB design for the NFTs so just added it as a column for now.
But I think I will create a specific table for this